docusaurus/docs/core-concepts/config-store/standard-config-type-host.v1.md

```diff
@@ -0,0 +1,45 @@
+# Standard Config Type host.v1
+
+A Config of Config Type `host.v1` is used by a Ziti tunneler application to forward packets to a destination server for a particular Ziti service.
+
+## Examples
+
+This Config forwards Ziti service traffic to a server known as `tcp:localhost:54321`. The server must be reachable at that destination from the network perspective of the device where the Ziti tunneler is running.
```

Comment:
I find it more natural to talk about intercept activity (and configurations) first, but that could just be me?